src/ReinforcementLearningDatasets/test/d4rl/d4rl_dataset.jl

```diff
@@ -0,0 +1,32 @@
+@testset "dataset_d4rl" begin
+    ds = dataset(
+        "hopper-medium-replay-v0";
+        style = SARTS,
+        rng = StableRNG(123),
+        is_shuffle = true,
+        batch_size = 256
+    )
+    n_s = 11
+    n_a = 3
+    N_samples = 200919
+
+    data_dict = ds.dataset
+
+    @test size(data_dict[:state]) == (n_s, N_samples)
+    @test size(data_dict[:action]) == (n_a, N_samples)
+    @test size(data_dict[:reward]) == (N_samples,)
+    @test size(data_dict[:terminal]) == (N_samples,)
+
+    i = 1
+
+    while i < 5
```

Comment:
Try `for batch in Iterators.take(ds, 5) ... end `

Reply:
It works.

Reply:
But I have made it as a for loop